### DISCUSSION TITLE
Property 'tagsKey' does not exist on type

### DISCUSSION THREAD

**Discussion by kaihendry** [KyleRoss/node-lambda-log#65] [Q&A]
How come I get these ts server warning/errors when using lambda-log?

<img src="https://s.natalian.org/2021-10-12/1634042444_1918x1047.png">

**Comment by KyleRoss** [comment]
@kaihendry I do not maintain the TypeScript Types for this package. The types were created by the community and are quite outdated at this point. Until those types are updated, you will receive these errors.

I'm working on v4.0.0 now which will be rewritten in TypeScript which will fix these issues.
